src/_actions/TradeActions.js

```diff
@@ -61,10 +61,24 @@ export const setQuickTradeField = (symbol, tradeType, field, value) => ({
     value,
 });
 
-export const createTrade = symbol => ({
-    type: types.CREATE_TRADE,
-    symbol,
-});
+export const createTrade = symbol =>
+    (dispatch, getState) => {
+        const activeIdx = getState().workspace.get('activeTradeIndex');
```

Comment:
Conceptually, createTrade does depends on all these states, I think it's fine, this is the way our UI works ~ =)

definitely possible to pass as params, this would means we shift the dependency to the Component itself instead of actionCreators.

Will do it